### DISCUSSION TITLE
关于标签的可选值讨论

### DISCUSSION THREAD

**Discussion by KimYangOfCat** [FEDarling/fe-translate-weekly#51] [项目讨论]
目前已有的标签值为：
```
tags: ['CSS','JavaScript','HTML','TypeScript','ECMAScript','Chrome','游览器','网络','React','Vue','webpack','babel','Vite','Node','HTTP','Rollup','Parcel','Svelte','Tool']
```
若上述标签可选项中没有你所翻译文章中相关的标签，请在下方评论留言，以讨论是否有增加相关标签的必要！

---留言格式：
```
相关文章：[标题](github url)，需要新增 `标签值`
```
示例：
```
相关文章：[Node.js 框架选择指南](https://github.com/FEDarling/weekly-tracker/blob/main/weeklys/node_weekly/419/Nodejs_Framework_Selection_Guide.md)，需要新增 `node`
```
效果：
相关文章：[Node.js 框架选择指南](https://github.com/FEDarling/weekly-tracker/blob/main/weeklys/node_weekly/419/Nodejs_Framework_Selection_Guide.md)，需要新增 `node`

---

开放此讨论的主要目的就是为了控制标签数量，避免重复相似的标签，比如：`VUE`、`Vue`、`Vue.js`等。所以希望大家都能自觉遵守此规定，不盲目添加未经讨论的标签值

**注意：标签值仅适用于精选文章，不适用于周刊，即所有名称为 README.md 的文档都不应该添加标签选项**

**Comment by daodaolee** [comment]
相关文章：[一个价值800万美元的娇羞按钮](https://github.com/FEDarling/weekly-tracker/blob/main/weeklys/mobile_dev_weekly/383/bashful_button.md)，需要新增 `移动端`

**Reply by KimYangOfCat** [reply]
目前的标签只有一类标签——技术名词，如果添加移动端的话，就新增了一类标签——平台名词？

**Reply by daodaolee** [reply]
倒也是...那 Mobile Dev Weekly 里的我改一下

**Reply by KimYangOfCat** [reply]
那就先不添加平台这一类的标签吧！

**Reply by daodaolee** [reply]
嗯呢
